index.js

```diff
@@ -73,28 +91,29 @@ nlf.find({ directory, production }, (err, npmPackages) => {
       builders.forEach(builder => {
         builder.addCached(name, version, lastVersionDefinition);
       });
-    } else if (_.get(npmPackage, 'licenseSources.package.sources', []).length > 0) {
-      const packageSources = _.get(npmPackage, 'licenseSources.package.sources', [])
-        .map(({ license }) => license)
-        .join(', ');
+    } else if (_.get(npmPackage, 'license', '').length > 0) {
+      const packagelicenseFile = `${_.get(npmPackage, 'license')}`;
 
       console.log(
-        chalk.red(`License source not found for ${packageID}, derived license types are: ${packageSources}.`)
+        chalk.red(`License source not found for ${packageID}, derived license types are: ${packagelicenseFile}.`)
       );
 
       builders.forEach(builder => {
         builder.addEmpty(name, version);
-      });
+      }); 
```

Comment:
Are you running with `prettier` here? It should have eliminated this extra space